pkg/tfgen/docs.go

```diff
@@ -1204,15 +1232,63 @@ func (g *Generator) convertHCL(hcl, path string) (string, string, error) {
 		err = convertHCL("go")
 	case Schema:
 		langs := []string{"typescript", "python", "csharp", "go"}
-		var anySucceeded bool = false
+		failedLangs := map[string]error{}
+		var passedLangs []string
+
 		for _, lang := range langs {
+			resultLenBefore := result.Len()
+
+			// The inner function convertHCL above will return an error in the event of a panic, but in the event of an
+			// error the function will simply not append any content to result and swallow the error. Therefore, we
```

Comment:
Feels like we could clarify this by changing `convertHCL` to return `(err error, fatal bool)` or something similar.

Reply:
I think I'm in favor of merging this as is and doing the refactor in a follow-up.

There's a lot of really hard to detect behavior in the inner function and we don't want to break existing behavior.

Reply:
`convertHCL` (inner edition) is something I'm eying for improvements in a future PR. If possible, I'd love for it to just return `(string, error)` and not be nested within another, identically named function. I'm sure there are reasons why it is the way it is, and I'm not sure I know them all, ergo the conservative approach - trying to deliver a steady-ish stream of value.